src/GovUk.Education.ExploreEducationStatistics.Content.Model/Extensions/FileExtensions.cs

```diff
@@ -80,7 +80,13 @@ public static string PublicPath(this File file, Guid releaseId)
 
         public static string ZipFileEntryName(this File file)
         {
-            return file.Type.GetEnumLabel() + "/" + file.Filename;
+            return file.Type switch
+            {
+                Ancillary => $"supporting-files/{file.Filename}",
+                Data => $"data/{file.Filename}",
+                DataGuidance => $"data-guidance/{file.Filename}",
```

Comment:
I only put `DataGuidance` in this sample code because I saw it in the screenshot on the ticket showing the contents of an existing zip file.

On review, I don't think `DataGuidance` needs to be here. It's not a file in the db like the `Ancillary` and `Data` files are, and gets added to the zip by another means:

```
                var entry = archive
                    .CreateEntry(FileType.DataGuidance.GetEnumLabel() + "/data-guidance.txt")
                    .SetUnixPermissions("664");
```